app/main.py

```diff
@@ -1,13 +1,10 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {"line": error["line_number"], "column": error["column_number"], "message": error["text"], "name": error["code"], "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {"errors": [{"line": er["line_number"], "column": er["column_number"], "message": er["text"], "name": er["code"], "source": "flake8"} for er in errors if er["filename"] == file_path], "path": file_path, "status": "failed"}
```

Comment:
Why do you not use you function for formatting single error? You duplicate your code. Remake.